src/AbstractUpdateAction.php

```diff
@@ -26,6 +28,8 @@ public function setId(?string $id = null): self
     }
 
     /**
+     * TODO: Allow also "int" as return type.
```

Comment:
I think here it should always be an `string`. Even if you set is a `int` it is converted to a string. That you can set it as an int is more a convenience for the dev but not for internal representation.